src/gt4py/next/program_processors/runners/dace_fieldview/gtir_dataflow.py

```diff
@@ -152,7 +180,7 @@ def connect(
         mx: dace.nodes.MapExit,
         result_node: dace.nodes.AccessNode,
         subset: sbs.Range,
-    ) -> None:
+    ) -> str | None:
```

Comment:
You should add a description that tells the user what it returns and why.
For my part, I do not really see why this function should return the local offset, it does not really make sense to me, at least at this point of the review.

Reply:
True, it is not needed. Some old code.